docs/podman-play-kube.1.md

```diff
@@ -20,7 +20,7 @@ kubernetes_input.yml
 the pod and containers described in the YAML.  The containers within the pod are then started and
 the ID of the new Pod is output.
 
-Ideally the input file would be one created by Podman.  This would guarantee a smooth import and expected results.
+Ideally the input file would be one created by Podman (see podman-generate(1)).  This would guarantee a smooth import and expected results.
```

Comment:
Shouldn't `podman-generate-kube(1)` also be referenced in the `SEE ALSO` section?

Reply:
Makes sense. I changed it directly to podman-generate-kube(1). Same for play. Thanks for the suggestion!